### PR TITLE
Update interop.md

### PR DESCRIPTION
**Pull request by Scottj1s**
elaborated com import requirements

### CODE REVIEW

File: docs/interop.md [hunk 1]
```diff
@@ -71,35 +71,39 @@ TInterop interop = objRef.AsInterface<TInterop>();
 ```
 
 ### Custom Marshaling
-When defining a [ComImport](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comimportattribute?view=netcore-3.1) interop interface, marshaling of WinRT types must be manual (not using the runtime).  
+**Note:** When defining a [ComImport](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comimportattribute?view=netcore-3.1) interop interface, WinRT parameters and return values must be passed by their ABI types, and marshaling must be done manually (not using the CLR).  For example, reference types like strings and interfaces must be passed as IntPtr.  Blittable value types can be passed directly. Non-blittable value types must have separate ABI and projected definitions, and marshaling between these values must be done manually.
```
Comment (manodasanW) [inline docs/interop.md:74]: Do we need our own documentation (list out) on what we consider a Blittable type or does .NET documentation cover it?

File: docs/interop.md [hunk 2]
```diff
@@ -71,35 +71,39 @@ TInterop interop = objRef.AsInterface<TInterop>();
...
 ```
 
 #### IInspectables
-The runtime no longer supports marshaling IInspectable-based ComImport interfaces. But these can be approximated with IUnknown:
+**Note:** The CLR no longer supports marshaling IInspectable-based ComImport interfaces. Casting to an IInspectable interface will succeed, but method calls will crash in the CLR. Instead, IInspectable interfaces can be approximated with IUnknown, by explicitly defining the GetIids, GetRuntimeClassName, and GetTrustLevel methods:
 ```csharp
 [ComImport]
 [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
 [Guid("68B3A2DF-8173-539F-B524-C8A2348F5AFB")]
 internal unsafe interface IServiceProviderInterop
```
Comment (manodasanW) [inline docs/interop.md:103]: Should we have a before and after?

File: docs/interop.md (hunk 1 above)
Comment (Scottj1s) [reply docs/interop.md]: added link to blittable .NET docs

File: docs/interop.md (hunk 2 above)
Comment (Scottj1s) [reply docs/interop.md:109]: added a bit of before/after code & docs

File: docs/interop.md [hunk 3]
```diff
@@ -108,8 +114,21 @@ internal unsafe interface IServiceProviderInterop
...
+
+// Cast a class or object to an interop interface using the .As<>() method described above
+var serviceProvider = someObject.As<IServiceProviderInterop>();
+IntPtr service;
+serviceProvider.GetService(IntPtr.Zero, out service);
+// See Create RCW section below for marshaling an object pointer from the ABI 
+
 ```
 
 ## Create RCW
```
Comment (manodasanW) [inline docs/interop.md:134]: Do we want to add details on when the respective FromAbi function should be used (i.e. ABI.Namespace.T.FromABI vs MarshalGeneric<T>.FromABI).   The other 2 are pretty obvious.

**Review by manodasanW — APPROVED** [review]

**Pull request merged**